ocs_sample_library_preview/Topics.py

```diff
@@ -0,0 +1,77 @@
+import json
+
+from .BaseClient import BaseClient
+from .OMF.Topic import Topic
+from .Securable import Securable
+
+
+class Topics(Securable, object):
+    """
+    Client for interacting with OCS Topics
+    This class does not represent the full functionality of the Topics route and is included to allow management of security on the route
+    """
+
+    def __init__(self, client: BaseClient):
+        """
+        Initializes the Topics client
+        :param client: This is the base client that is used to make REST calls
+        """
+        super().__init__(client=client, collection='Topics')
+
+        self.__base_client = client
+
+        self.__setPathAndQueryTemplates()
+
+    def getTopicById(self, namespace_id: str, topic_id: str) -> Topic:
+        """
+        Returns the specified Topic
+        :param namespace_id: The namespace identifier
+        :param topic_id: The topic identifier
+        """
+        if namespace_id is None:
+            raise TypeError
+        if topic_id is None:
+            raise TypeError
+
+        response = self.__base_client.request('get', self.__topic_path.format(
+            namespace_id=namespace_id, topic_id=topic_id))
+        self.__base_client.checkResponse(
+            response, f'Failed to get Topic, {topic_id}.')
+
+        result = Topic.fromJson(response.json())
+        return result
+
+    def getTopics(self, namespace_id: str, query: str = '', skip: int = 0, count: int = 100,
```

Comment:
same comment about the query, count, and skip. If I'm doing something wrong, I retract all of these comments :|

Reply:
Same comment as for Subscriptions